experimental/dds/tree2/src/shared-tree/schematizedTree.ts

```diff
@@ -5,94 +5,108 @@
 
 import { assert, unreachableCase } from "@fluidframework/common-utils";
 import {
-	schemaDataIsEmpty,
 	AllowedUpdateType,
 	Compatibility,
 	SimpleObservingDependent,
 	SchemaData,
+	StoredSchemaRepository,
 	ITreeCursorSynchronous,
+	schemaDataIsEmpty,
 } from "../core";
 import {
-	ViewSchema,
 	defaultSchemaPolicy,
 	FieldKinds,
 	allowsRepoSuperset,
 	TypedSchemaCollection,
 	SchemaAware,
 	FieldSchema,
+	ViewSchema,
 } from "../feature-libraries";
 import { fail } from "../util";
-import { ISharedTreeView } from "./sharedTreeView";
+import { ISubscribable } from "../events";
+import { ViewEvents } from "./sharedTreeView";
+
+/**
+ * Modify `storedSchema` and invoke `setInitialTree` when its time to set the tree content.
+ *
+ * Requires `storedSchema` to be in its default/empty state.
+ *
+ * This is done in such a way that if the content that the content (implicitly assumed to start empty)
```

Comment:
This should make it make more sense
```suggestion
 * This is done in such a way that if the content (implicitly assumed to start empty)
```